transport/p2p/src/swarm.rs

```diff
@@ -117,7 +117,7 @@ async fn build_p2p_network(
             )
             .with_max_negotiating_inbound_streams(
                 std::env::var("HOPR_INTERNAL_LIBP2P_MAX_NEGOTIATING_INBOUND_STREAM_COUNT")
-                    .map(|v| v.trim().parse::<usize>().unwrap_or(128))
+                    .and_then(|v| v.parse::<usize>().map_err(|_e| std::env::VarError::NotPresent))
```

Comment:
No default value here?

Reply:
There will be a default inherited in the next line. I just rewrote it to be compliant with how the rest of the env vars are taken care of.